### PR TITLE
tests: add helper for finding target volume path inside guest

### PR DESCRIPTION
**Pull request by akalenyu**
### What this PR does
We have usage of this scattered around, let's unify and use it more instead of guessing the device name.

Fixes #

### Why we need it and why it was done in this way
The following tradeoffs were made:

The following alternatives were considered:

Links to places where the discussion took place: 

### Checklist

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

### Release note

```release-note
NONE
```

### CODE REVIEW

File: tests/libstorage/datavolume.go [hunk 1]
```diff
@@ -35,7 +35,6 @@ import (
 	v12 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/rand"
 
-	v1 "kubevirt.io/api/core/v1"
```
Comment (EdDev) [inline tests/libstorage/datavolume.go:38]: You better drop the v13 instead.

File: tests/libstorage/vmi.go [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+package libstorage
+
+import (
+	"fmt"
+
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func GetVolumeTargetPath(vmi *v1.VirtualMachineInstance, volumeName string) string {
```
Comment (EdDev) [inline tests/libstorage/vmi.go:28]: nit: Personally I prefer `Lookup` over `Get`. Getters have none to minimal logic, but here we have more than that.

**Review by ShellyKa13 — APPROVED** [review]
Nice!
/lgtm

**Review by EdDev — APPROVED** [review]
Thank you!

**Review by ShellyKa13 — APPROVED** [review]
/lgtm

**Review by EdDev — APPROVED** [review]
Great, thanks!

**Comment by ShellyKa13** [comment]
@EdDev it needs the "approve" label

**Comment by EdDev** [comment]
> @EdDev it needs the "approve" label

I have that power only on the network folders and `libvmi`.
@enp0s3 , can you please have a look?

**Pull request merged**